addons/website_slides/models/slide_question.py

```diff
@@ -56,3 +56,5 @@ class SlideAnswer(models.Model):
     question_id = fields.Many2one('slide.question', string="Question", required=True, ondelete='cascade')
     text_value = fields.Char("Answer", required=True, translate=True)
     is_correct = fields.Boolean("Is correct answer")
+    # TODO: Should it be required ? Translatable ? Maybe check for one question if every answers has a comment ?
+    comment = fields.Text("Details")
```

Comment:
Have you been able to figure out your "TODO" question?

IMO it should definitely be `translatable`, but not `required`.

Also, I'm not sure "Details" is a very good label.
Maybe "Comment" with a `help="This comment will be displayed to the user if he selects this answer"`

Reply:
Actually I totally forgot about this TODO :(

But I asked sbu if all answers should have a comment if we add it to one answer and the answer was no.